app.py

```diff
@@ -13,47 +13,80 @@
 MICROSOFT_API_KEY = os.environ['MICROSOFT_API_KEY']
 MICROSOFT_ENDPOINT = os.environ['MICROSOFT_ENDPOINT']
 SCAN_FREQUENCY = os.environ['SCAN_FREQUENCY'] if 'SCAN_FREQUENCY' in os.environ else None
+SCAN_METHOD = os.environ['SCAN_METHOD'].lower() # checkbox or createtime
 
+# This is a dictionary that contains the headers that will be sent with every request to the Notion
+# API.
 HEADERS = {
     "Authorization": "Bearer " + NOTION_TOKEN,
     "Content-Type": "application/json",
     "Notion-Version": "2022-06-28"
 }
 
+def get_scan_requestBody():
```

Comment:
Suggesting to make this more 'pythonic' by changing the camelCase into a snake_case (eg get_scan_request_body)

Reply:
Yes I don't develop in python anymore but in nodejs and so I use camelCase instead.
So i've working hard to use snake_case but sometime my daemons were to strong :)